db/migrate/20161206130507_create_dispatchers.rb

```diff
@@ -0,0 +1,14 @@
+class CreateDispatchers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :dispatchers do |t|
+      t.string :email
```

Comment:
@DmitrS so what about to add validations in the database too?
http://stackoverflow.com/questions/13122791/rails-validation-in-model-vs-migration